src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,14 @@
+spring:
+  cloud:
+    propertiesvolume:
+      paths: /mnt/secrets/cmc
+      prefixed: false
+      aliases:
+        claim-store-s2s-secret: IDAM_S2S_AUTH_TOTP_SECRET
+        notify-api-key: GOV_NOTIFY_API_KEY
+        citizen-oauth-client-secret: oauth2.client.secret
+        anonymous-caseworker-username: IDAM_CASEWORKER_ANONYMOUS_USERNAME
+        anonymous-caseworker-password: IDAM_CASEWORKER_ANONYMOUS_PASSWORD
+        system-update-username: IDAM_CASEWORKER_SYSTEM_USERNAME
+        system-update-password: IDAM_CASEWORKER_SYSTEM_PASSWORD
+        AppInsightsInstrumentationKey: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
this is the wrong property on java logging 5.0.1
https://github.com/hmcts/java-logging/tree/master/java-logging-appinsights#provide-instrumentation-key